webhooks/webhooks.helper.ts

```diff
@@ -0,0 +1,300 @@
+import { StepLogger } from '../../../../core/logger/step-logger';
+import { ExpectationHelper } from '../../../components/misc-utils/expectation-helper';
+import { CommonPageHelper } from '../common/common-page.helper';
+import { PageHelper } from '../../../components/html/page-helper';
+import { TextBoxHelper } from '../../../components/html/textbox-helper';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
import { TextBoxHelper } from "../../../components/html/textbox-helper";
```